src/libraries/System.Net.Http/tests/UnitTests/Headers/CacheControlHeaderValueTest.cs

```diff
@@ -655,6 +671,15 @@ public void TryParse_SetOfInvalidValueStrings_ReturnsFalse()
             CheckInvalidTryParse("\u4F1A", 0);
         }
 
+        [Fact]
+        public void TryParse_SetOfInvalidValueStrings_ReturnsEmptyValue()
+        {
+            bool parsed = CacheControlHeaderValue.TryParse("no-cache,=", out CacheControlHeaderValue parsedValue);
+            Assert.True(parsed);
+            Assert.IsType<CacheControlHeaderValue>(parsedValue);
+            Assert.False(parsedValue is null);
+        }
+
```

Comment:
```suggestion
```

We already have tests for "valid + empty" [above](https://github.com/dotnet/runtime/blob/f7e53268815bf1a7ffbb6d9f1f214e46f865fcad/src/libraries/System.Net.Http/tests/UnitTests/Headers/CacheControlHeaderValueTest.cs#L654).